src/DssSpell.sol

```diff
@@ -44,10 +73,79 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
         // Includes changes from the DssSpellCollateralOnboardingAction
         // onboardNewCollaterals();
 
-        // Deauthorize the existing lerp to prevent additional overwrites of hump.
-        DssExecLib.deauthorize(DssExecLib.vow(), SB_LERP);
+        // Revoking Content Production Budget (MKT-001)
+        // https://mips.makerdao.com/mips/details/MIP40c3SP49
+        DssVestLike(MCD_VEST_DAI).yank(20);
+
+
+        //Core Unit DAI Budget Transfers
+        // https://mips.makerdao.com/mips/details/MIP40c3SP47
+        DssExecLib.sendPaymentFromSurplusBuffer(SNE_001_WALLET, 229_792);
+        // https://mips.makerdao.com/mips/details/MIP40c3SP53
+        DssExecLib.sendPaymentFromSurplusBuffer(TECH_001_WALLET, 1_069_250);
+        // https://mips.makerdao.com/mips/details/MIP40c3SP45
+        DssExecLib.sendPaymentFromSurplusBuffer(ORA_001_GAS, 6_966_070);
+        // https://mips.makerdao.com/mips/details/MIP40c3SP45
+        DssExecLib.sendPaymentFromSurplusBuffer(ORA_001_GAS_EMERGENCY, 1_805_407);
+
+
+        // Core Unit DAI Budget Streams
+        // https://mips.makerdao.com/mips/details/MIP40c3SP52
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(DUX_001_WALLET,   1_934_300 * WAD, FEB_01_2022, JAN_31_2023 - FEB_01_2022,            0, address(0))
+        );
+        // https://mips.makerdao.com/mips/details/MIP40c3SP55
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(SES_001_WALLET,   5_844_444 * WAD, FEB_01_2022, JAN_31_2023 - FEB_01_2022,            0, address(0))
+        );
+        // https://mips.makerdao.com/mips/details/MIP40c3SP47
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(SNE_001_WALLET,     257_500 * WAD, FEB_01_2022, JUL_31_2023 - FEB_01_2022,            0, address(0))
+        );
+        // https://mips.makerdao.com/mips/details/MIP40c3SP53
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(TECH_001_WALLET,  2_566_200 * WAD, FEB_01_2022, JAN_31_2023 - FEB_01_2022,            0, address(0))
+        );
+        // https://mips.makerdao.com/mips/details/MIP40c3SP46
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(SF_001_WALLET,      494_502 * WAD, FEB_01_2022, JUL_31_2023 - FEB_01_2022,            0, address(0))
+        );
+        // - Forum Post TODO
```

Comment:
Maybe use this:
https://forum.makerdao.com/t/transition-of-the-rwf-cu-multisig/12801/2